index.html

```diff
@@ -0,0 +1,262 @@
+            </div>
+            <div class="col-md">
+              <div class="card bg-secondary text-light">
+                <div class="card-body text-center">
+                  <div class="h1 mb-3">
+                    <i class="bi bi-people"></i>
+                  </div>
+                  <h3 class="card-title mb-3">LH Ignite</h3>
+                  <p class="card-text">
+                    Get opportunity to network with LH affiliates around the world also after the summit. Thanks to LH ...
+                  </p>
+                  <a href="#" class="btn btn-primary">Read More</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="mt-5">
+            <a href="#" class="see-program text-white mt-5"> SEE THE WHOLE PROGRAM</a>
+          </div>
+        </div>
+      </div>
+
+    </section>
+
+           <!-- instructors section -->
+           <section id="instructors" class="instructors p-5 text-dark">
+            <div class="container">
+              <h2 class="text-center text-dark">Featured Speakers</h2>
+              <p class="lead text-center  mb-5">
+                Our invited speakers all have 5+ years working as a web developer in the
+                industry
+              </p>
+              <div class="row g-2 speakers">
+                </div>
+              </div>
+          </section>
+
+          <!-- Newsletter -->
+    <section class="bg-dark text-light p-5 text-center">
+      <div class="container">
+        <h3 class="lead fw-bold display-5">Partners</h3>
+        <div class="row partners-list-section">
+          <div class="col-md">
+            <a href="#" class="partner-logo"><img src="./images/icons/google.png" alt="google"></a>
+          </div>
+          <div class="col-md">
+            <a href="#" class="partner-logo"><img src="./images/icons/partner1.png" alt="hub"></a>
+          </div>
+          <div class="col-md">
+            <a href="#" class="partner-logo"><img src="./images/icons/ictauthority.png" alt="ictAuthority"></a>
+          </div>
+          <div class="col-md">
+            <a href="#" class="partner-logo"><img src="./images/icons/oracle.png" alt="oracle"></a>
+          </div>
+          <div class="col-md">
+            <a href="#" class="partner-logo"><img src="./images/icons/azure.png" alt="azure"></a>
+          </div>
+        </div>
+      </div>
+    </section>
+        </main>
+    <!-- Footer -->
+    <footer class="p-5 bg-light text-dark text-center position-relative">
+      <div class="container">
+          <div class="d-flex text-center footer-text">
+           <a href="#"><img class="p-5" src="./images/icons/partner2.png" alt="lawyers hub"></a>
+           <p class="px-5 pt-5">Africa's inaugural LawTech Conference, Hackathon, Awards & Institute
+            <br><span class="fw-bold">Lawyers Hub | tech for justice © 2022</span>
+           </p>
+            
+          </div>
+      </div>
+    </footer>   
```

Comment:
- [x] According to the original design, the `partners` and `footer` sections should not be shown in the **mobile version**, the last section for the `mobile version` should be the speakers (or the artists in your project). Please, kindly address this issue in the mobile version in order to match the design :smiley: :raised_hands:

| Original design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147131660-823d562f-6346-4262-9854-f7292853388d.png) | ![image](https://user-images.githubusercontent.com/45231276/181269454-72db69bc-e0c4-4551-97a3-f3ba8eeda0c5.PNG) |